test/actions/server_test.js

```diff
@@ -23,6 +23,23 @@ import {
   NOTIFICATION_ADD
 } from "../../formbuilder/actions/notifications";
 
+function getErrorDispatch(done) {
+  return (func) => {
+    if (typeof func === "function") {
```

Comment:
Nit: returning early when `func` is not a function might ease legibility a little